packages/edit-site/src/components/page-patterns/use-patterns.js

```diff
@@ -35,7 +36,7 @@ const templatePartToPattern = ( templatePart ) => ( {
 	keywords: templatePart.keywords || [],
 	id: createTemplatePartId( templatePart.theme, templatePart.slug ),
 	name: createTemplatePartId( templatePart.theme, templatePart.slug ),
-	title: templatePart.title.rendered,
+	title: decodeEntities( templatePart.title.rendered ),
```

Comment:
Decoding here eliminates the need for separate decoding in the lower-layer components. This is consistent with the fact that [`reusableBlockToPattern` in the same file uses `title.raw` as the value of the title property](https://github.com/WordPress/gutenberg/blob/aaf6f8f5e044624a9ff98dbd5acd6c38749e297b/packages/edit-site/src/components/page-patterns/use-patterns.js#L154).